qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -169,33 +169,41 @@ def _check_if_stopped(self) -> None:
 
         Raises:
             SuperstaqUnsuccessfulJobException: If the job has been cancelled or has
-        failed.
+                failed.
```

Comment:
nit: it looks like this would fit on one line

Reply:
You're right, fixed!